lib/utils/__tests__/isStandardSyntaxDeclaration.test.js

```diff
@@ -1,241 +1,179 @@
-		scssDecls('$var: b', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('$var: b'))).toBeFalsy();
 	});
 
 	it('scss var within namespace', () => {
-		scssDecls('namespace.$var: b', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('namespace.$var: b'))).toBeFalsy();
 	});
 
 	it('nested scss var within namespace', () => {
-		scssDecls('a { namespace.$var: b }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('a { namespace.$var: b }'))).toBeFalsy();
 	});
 
 	it('sass list', () => {
-		sassDecls('$list: (key: value, key2: value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('$list: (key: value, key2: value2)'))).toBeFalsy();
 	});
 
 	it('scss list', () => {
-		scssDecls('$list: (key: value, key2: value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('$list: (key: value, key2: value2)'))).toBeFalsy();
 	});
 
 	it('sass map', () => {
-		sassDecls('$map: (value, value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('$map: (value, value2)'))).toBeFalsy();
 	});
 
 	it('scss map', () => {
-		scssDecls('$map: (value, value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('$map: (value, value2)'))).toBeFalsy();
 	});
 
 	it('nested sass var', () => {
-		sassDecls('a\n\t$var: b', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('a\n\t$var: b'))).toBeFalsy();
 	});
 
 	it('nested scss var', () => {
-		scssDecls('a { $var: b }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(scssDecl('a { $var: b }'))).toBeFalsy();
 	});
 
 	it('nested sass list', () => {
-		sassDecls('a\n\t$list: (key: value, key2: value2)', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(
+			isStandardSyntaxDeclaration(sassDecl('a\n\t$list: (key: value, key2: value2)')),
+		).toBeFalsy();
 	});
 
 	it('nested scss list', () => {
-		scssDecls('a { $list: (key: value, key2: value2) }', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(
+			isStandardSyntaxDeclaration(scssDecl('a { $list: (key: value, key2: value2) }')),
+		).toBeFalsy();
 	});
 
 	it('sass nested property', () => {
-		sassDecls('a\n\tborder:\n\t\tstyle: solid\n\t\tcolor: red', (decl) => {
-			expect(isStandardSyntaxDeclaration(decl)).toBeFalsy();
-		});
+		expect(isStandardSyntaxDeclaration(sassDecl('a\n\tborder:\n\t\tstyle: solid'))).toBeFalsy();
```

Comment:
[note] Note that this case removes the `color: red` declaration.

```diff
-a\n\tborder:\n\t\tstyle: solid\n\t\tcolor: red
+a\n\tborder:\n\t\tstyle: solid
```

Because:

- `sassDecl()` returns only one declaration
- it is needless to test two declarations